src/main/java/seedu/address/logic/parser/contact/ContactCommandsParser.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.parser.contact;
+
+import static seedu.address.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.contact.AddCommand;
+import seedu.address.logic.commands.contact.DeleteCommand;
+import seedu.address.logic.commands.contact.EditCommand;
+import seedu.address.logic.commands.contact.FindCommand;
+import seedu.address.logic.commands.contact.ListCommand;
+import seedu.address.logic.parser.GroupCommandsParser;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class ContactCommandsParser implements GroupCommandsParser {
+
+    @Override
```

Comment:
Maybe it would be better if we add comments along the way as we write code? Just a suggestion haha.

Reply:
Thanks for the suggestion, I will add them now!